cpp/src/io/orc/stripe_data.cu

```diff
@@ -1403,7 +1403,7 @@ __global__ void __launch_bounds__(block_size)
   }
   __syncthreads();
   // Struct doesn't have any data in itself, so skip
-  const bool is_valid       = s->chunk.type_kind != STRUCT;
+  bool const is_valid       = s->chunk.type_kind != STRUCT;
   const size_t max_num_rows = s->chunk.column_num_rows;
```

Comment:
here as well. Are there known cases where clang-format does not apply the const style?